ravenpy/models/base.py

```diff
@@ -398,16 +398,14 @@ def run(self, ts, overwrite=False, **kwds):
 
         """
         if isinstance(ts, (six.string_types, Path)):
-            ts = [
-                ts,
-            ]
+            ts = [ts]
 
         # Case for potentially parallel parameters
         pdict = {}
         for p in self._parallel_parameters:
             a = kwds.pop(p, None)
 
-            if a is not None and p in ["params", "basin_state", "hru_state"]:
+            if a is not None and p in ["params"]:
```

Comment:
Won't this break support for parallel runs on multiple hru_states ? This functionality going to be needed for assimilation.

Reply:
I realize this is coming from another branch. Please ignore.

Reply:
Not sure where this is coming from, this PR is bound to be very complex (I'm merging my routing PR to avoid too many future conflicts)